PopulateTools/gobierto
# 261 events timeline limit

related with [#261](/PopulateTools/issues/issues/261)

### What does this PR do?
* Limits to 5 the activities list appearing in index of scopes and issues
* Removes the hour present in events lists

### How should this be manually tested?
Visit An issue or scope index page and check the second column for the first point and the first column for the second

### Does this PR changes any configuration file?
No
